Source/SA/RFmx SpecAn/RFmxSpecAn.cs

```diff
@@ -44,33 +40,313 @@ public struct AutoLevelConfiguration
+                };
+        
+            }
+        }
+        public struct AcpOffsetChannelConfiguration
+        {
+            public double IntegrationBandwidth_Hz;
+            public RFmxSpecAnMXAcpOffsetEnabled OffsetEnabled;
+            public double OffsetFrequency_Hz;
+            public RFmxSpecAnMXAcpOffsetSideband OffsetSideBand;
+            public RFmxSpecAnMXAcpOffsetPowerReferenceCarrier OffsetPowerReferenceCarrier;
+            public int OffsetPowerReferenceSpecificIndex;
+            public double RelativeAttenuation_dB;
+            public RFmxSpecAnMXAcpOffsetRrcFilterEnabled RrcFilterEnabled;
+            public double RrcFilterAlpha;
+
+            public static AcpOffsetChannelConfiguration GetDefault()
+            {
+                return new AcpOffsetChannelConfiguration
+                {
+                    IntegrationBandwidth_Hz = 18e6,
+                    OffsetEnabled = RFmxSpecAnMXAcpOffsetEnabled.True,
+                    OffsetFrequency_Hz = 20e6,
+                    OffsetSideBand = RFmxSpecAnMXAcpOffsetSideband.Both,
+                    OffsetPowerReferenceCarrier = RFmxSpecAnMXAcpOffsetPowerReferenceCarrier.Closest,
+                    OffsetPowerReferenceSpecificIndex = 0,
+                    RelativeAttenuation_dB = 0,
+                    RrcFilterEnabled = RFmxSpecAnMXAcpOffsetRrcFilterEnabled.False,
+                    RrcFilterAlpha = 0.22
+                };
+            }
+        }
+
+        public struct AmpmConfiguration
+        {
+            public double MeasurementInterval_s;
+            public double DutInputPower_dBm;
+            public ComplexWaveform<ComplexSingle> ReferenceWaveform;
+            public RFmxSpecAnMXAmpmReferenceWaveformIdleDurationPresent IdleDurationPresent;
+            public RFmxSpecAnMXAmpmSignalType SignalType;
+
+
+            public static AmpmConfiguration GetDefault()
+            {
+                return new AmpmConfiguration
+                {
+                    MeasurementInterval_s = 100e-6,
+                    DutInputPower_dBm = -20,
+                    ReferenceWaveform = new ComplexWaveform<ComplexSingle>(0),
+                    IdleDurationPresent = RFmxSpecAnMXAmpmReferenceWaveformIdleDurationPresent.False,
+                    SignalType = RFmxSpecAnMXAmpmSignalType.Modulated
+                };
+
+            }
+        }
+
+        public struct TxpResults
+        {
+            public double AverageMeanPower_dBm;
+            public double PeakToAverageRatio_dB;
+            public double MaximumPower_dBm;
+            public double MinimumPower_dBm;
+        }
+        public struct AcpOffsetResults
+        {
+            public double LowerAbsolutePower_dBm;
+            public double LowerRelativePower_dB;
+            public double UpperAbsolutePower_dBm;
+            public double UpperRelativePower_dB;
+            public double OffsetFrequency_Hz;
+            public double OffsetIntegrationBandwidth_Hz;

```

Comment:
I think i'd prefer to remove the "Offset" prefix. Will have to do this for LTE and NR as well. To be consistent.